### PR TITLE
rhacs: fix s3 compat endpoint docs

### PR DESCRIPTION
**Pull request by stehessel**
Version(s):

Issue: [ROX-25216](https://issues.redhat.com/browse/ROX-25216)

[Link to docs preview
]([78980--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-s3-api-compatible-services.html](https://78980--ocpdocs-pr.netlify.app/openshift-acs/latest/integration/integrate-with-s3-api-compatible-services.html))

QE review: (**ACS has no QE, reviewed/ack'd by SME**)
- [x] QE has approved this change.

Additional information: Small correction to the S3 compatible backups after we changed the endpoint validation logic slightly.

### CODE REVIEW

**Review by lvalerom — APPROVED** [review]
LGTM

File: modules/s3-api-compatible-services-configuring-acs.adoc [hunk 1]
```diff
@@ -21,7 +21,7 @@ To create a new bucket with required permissions, see your S3 provider documenta
 . For *Schedule*, select the backup frequency as daily or weekly, and select the time to run the backup process.
 . Enter the *Bucket* name where you want to store the backup.
 . Optionally, enter an *Object Prefix* if you want to save the backups in a specific folder structure.
-. Enter the *Endpoint* under which the S3 compatible service is reached. It must be reachable via https.
+. Enter the *Endpoint* under which the S3 compatible service is reachable. Defaults to https if the scheme is left out.
```
Comment (kcarmichael08) [inline modules/s3-api-compatible-services-configuring-acs.adoc:24]: I'm kind of unsure of what this means - does it mean that RHACS attempts to use https for the connection to the endpoint if no connection type is specified? (I'm not sure what "scheme" means in this context.)

**Review by kcarmichael08 — COMMENTED** [review]
one question about wording

File: modules/s3-api-compatible-services-configuring-acs.adoc (hunk 1 above)
Comment (stehessel) [reply modules/s3-api-compatible-services-configuring-acs.adoc:24]: Yes, in `https://play.min.io` the scheme is `https`. It means that if you enter `play.min.io`, the backend will change this to `https://play.min.io`.

Comment (kcarmichael08) [inline modules/s3-api-compatible-services-configuring-acs.adoc:24]:
```suggestion
. Enter the *Endpoint* under which the S3 compatible service is reachable. If no scheme is specified, the default, `https`, is used.
```
Would this be accurate?

Comment (stehessel) [reply modules/s3-api-compatible-services-configuring-acs.adoc:24]: Yes 👍

**Pull request merged**